src/component/admin/apiToken/ApiTokenForm/SelectProjectInput.tsx

```diff
@@ -0,0 +1,150 @@
+                                if (e.target.checked) {
+                                    selectWildcard(true);
+                                    onChange([ALL_PROJECTS]);
+                                } else {
+                                    selectWildcard(false);
+                                    onChange(
+                                        projects.includes(ALL_PROJECTS)
+                                            ? []
+                                            : projects
+                                    );
+                                }
+                            }}
+                        />
+                    }
+                    label="ALL current and future projects"
+                />
+            </Box>
+            <Autocomplete
+                disabled={disabled || isWildcardSelected}
+                multiple
+                limitTags={2}
+                options={options}
+                disableCloseOnSelect
+                getOptionLabel={({ label }: IAutocompleteBoxOption) => label}
+                groupBy={() => 'Select/Deselect all'}
+                renderGroup={({ key, children }) => (
+                    <Fragment key={key}>
+                        {selectAllButton}
+                        {children}
+                    </Fragment>
+                )}
+                fullWidth
+                PaperComponent={CustomPaper}
+                renderOption={(
+                    option: IAutocompleteBoxOption,
+                    { selected }
+                ) => (
+                    <>
+                        <Checkbox
+                            icon={icon}
+                            checkedIcon={checkedIcon}
+                            checked={selected}
+                            className={styles.selectOptionCheckbox}
+                        />
+                        {option.label}
+                    </>
+                )}
+                renderInput={params => (
+                    <TextField
+                        {...params}
+                        error={!!error}
+                        helperText={error}
+                        variant="outlined"
+                        label="Projects"
+                        placeholder="Select one or more projects"
+                        onFocus={onFocus}
+                    />
+                )}
+                value={
+                    isWildcardSelected || disabled
+                        ? options
+                        : options.filter((option: IAutocompleteBoxOption) =>
+                              projects.includes(option.value)
+                          )
+                }
+                onChange={(_, input) => {
+                    const state = input.map(({ value }) => value);
+                    setProjects(state);
+                    onChange(state);
+                }}
+            />
+        </Box>
+    );
+};
+
+export default SelectProjectInput;
```

Comment:
We're aiming to remove all default exports, so would export `SelectProjectInput` directly instead.